docs/source/en/_toctree.yml

```diff
@@ -19,7 +19,10 @@
     title: Train a diffusion model
   - local: tutorials/using_peft_for_inference
     title: Inference with PEFT
-  title: Tutorials
+  - local: tutorials/inference_with_lcm_lora
```

Comment:
I think these would be more appropriate in the "Specific pipeline examples" section